microvenv.py

```diff
@@ -76,12 +76,9 @@ def create(env_dir=".venv"):
 
 
 if __name__ == "__main__":
-    if len(sys.argv) > 2:
-        print("Usage: microvenv.py [env_dir='.venv']", file=sys.stderr)
-        sys.exit(1)
-    try:
-        env_dir = sys.argv[1]
-    except IndexError:
-        env_dir = ".venv"
+    import argparse
 
-    create(env_dir)
+    parser = argparse.ArgumentParser()
+    parser.add_argument("env_dir", default=".venv", nargs="?", help="default: '.venv'")
```

Comment:
Shouldn't `help` argument be more descriptive? Such as the removed line in the last version.
```print("Usage: microvenv.py [env_dir='.venv']", file=sys.stderr)```

Reply:
I specifically made sure that it contains the same information. 😊

```bash
$ py microvenv.py -h
usage: microvenv.py [-h] [env_dir]

positional arguments:
  env_dir     default: '.venv'
```